lib/journaled.rb

```diff
@@ -4,9 +4,11 @@
 require "request_store"
 
 require "journaled/engine"
+require "journaled/job_priority"
```

Comment:
I moved this from `app/models` to `lib` and therefor need to require it manually. I did this so that I could use it to define the default in this file